tests/keras/backend/backend_test.py

```diff
@@ -10,6 +10,9 @@
 from keras.backend import cntk_backend as KC
 
 BACKENDS = [KTH, KTF, KC]
+# for special test cases of CNTK which treat dynamic axis
+KCD = 'CNTK_dynamic_axis'
+KCDT = 'CNTK_dynamic_axis_two_tensor'
```

Comment:
The author of the current CNTK test functions wrote the separate three functions:
- `cntk_check_single_tensor_operation`
- `cntk_check_two_tensor_operation_with_batch`
- `cntk_check_two_tensor_operation_with_single_batch`

I just factorized over these functions and integrated into `check_*_tensor_operation`. In my understanding, there are two types for CNTK placeholder creation: `KC.placeholder(shape)` and `KC.placeholder(ndim=len(shape))`. I used the flags `KCD` and `KCDT` to cover up the two types. The abbreviations mean `C`NTK, `D`ynamic_axis, and `T`wo_tensor. Reviewing back now, they seem to be not good, so I changed some names including them:
- `cntk_func_for_dynamic_axis` -> `cntk_func_single_tensor` and `cntk_func_two_tensor`
- `KCD` -> `KCSD`
- `KCDT` -> `KCTD`